composer.json

```diff
@@ -25,7 +25,7 @@
         "jms/di-extra-bundle": "~1.4.0",
         "surfnet/stepup-middleware-client-bundle": "^1.3",
```

Comment:
This needs to be updated and released (and updated as a dependency!) before this can be merged.